Uni_life_tracker/app/src/main/res/layout/activity_expenses_list.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                                     android:layout_width="match_parent"
+                                                   xmlns:app="http://schemas.android.com/apk/res-auto"
+                                                   android:layout_height="match_parent">
+
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/Expenses"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.025"
+            app:layout_constraintHorizontal_bias="0.376"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="20sp"
+            android:textColor="@color/black"
+            android:id="@+id/expenses_title"/>
+
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/month"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.141"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="18sp"
+            android:textColor="#070707"
+            android:id="@+id/expense_month"
+            app:layout_constraintHorizontal_bias="0.747"/>
+
+
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/list_of_expenses"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.264"
+            app:layout_constraintHorizontal_bias="0.142"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="18sp"
+            android:textColor="#070707"
+            android:id="@+id/expense_list"/>
+    <TextView
```

Comment:
instead of hardcoding this part, I think it'd be better to use RecyclerView, so you can repeat the expense "item" with this view rather than repeating these elements over again?